1.6/Dockerfile

```diff
@@ -0,0 +1,117 @@
+FROM openshift/base-centos7
+
+ARG ERLANG_VERSION=20.0
```

Comment:
That's an old version

Reply:
@maennchen What version would you prefer for 1.6?

Reply:
The newest stable release, `20.3`

Reply:
Done in 21ce772